website/docs/guides/scripts.md

```diff
@@ -163,6 +163,32 @@ world
 
 </ChainedSnippets>
 
+### Define source files in using directives
+
+You can also add source files with the using directive `//> using file` in Scala Scripts:
+
+```scala title=main.sc
+//> using file "Utils.scala" 
+
+println(Utils.message)
+```
+```scala title=Utils.scala
+object Utils {
+  val message = "Hello World"
+}
+```
+
+`scala-cli` compiles and adds to the class path source `Utils.scala`.
```

Comment:
NIT Phrasing (the source itself isn't added to the class path per se, more its… "build products" I'd say)
```suggestion
`scala-cli` takes into account and compiles `Utils.scala`.
```